Gemfile

```diff
@@ -7,7 +7,9 @@ gem 'rake'
 gem 'yard'
 gem 'rspec'
 
-gem "psych", '<= 5.2.0'
+gem 'fiddle'
```

Comment:
Can you add to the PR which error was related to Fiddle?

Reply:
https://github.com/pry/pry/actions/runs/10832077586/job/30058679245#step:7:76

Reply:
As the 'fiddle' warnings are coming from Reline, I don't think we should fix by adding this dependency here

https://github.com/ruby/reline/blob/5353924f323d4afa8c285997c5d950c62c4e2c1d/lib/reline/terminfo.rb#L1-L15

And it will also work in the future when Fiddle is not around.. 

The most recent version of reline fixed this, so I made another [PR](https://github.com/pry/pry/pull/2320) to ensure the CI runs with an recent release of Reline.